src/components/controls/variables.scss

```diff
@@ -1 +1,4 @@
 $border-width: 1px;
+// Safari on iOS zoom page if input font-size is smaller than 16px
+// https://css-tricks.com/16px-or-larger-text-prevents-ios-form-zoom/
+$mobile-safari-font-size: 16px;
```

Comment:
I suggest u to rename this to $mobile-font-size case of it uses not only for safari styles

Reply:
May be `$mobile-min-font-size` then?

Reply:
But we do not have a max size or other size at all

Reply:
But we do have minimal size, it's not just any mobile font size, it is a minimal possible size for Safari

Reply:
ok. It's not a public api